lms/static/styles/moodle_pages/moodle_pages.scss

```diff
@@ -0,0 +1,226 @@
+body {
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto,
+    'Helvetica Neue', Arial, 'Noto Sans', 'Liberation Sans', sans-serif,
+    'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol', 'Noto Color Emoji';
+  font-size: 0.9375rem;
```

Comment:
This value looks like something that was produced by a SASS compiler or something similar. Have you got a link for the original source of truth in the Moodle source repository?

Reply:
Good point. Let me check that.

Reply:
It's a hardcoded value. In some changelog they mention the use of this value after updating to bootstrap 4, as it translates to ~15px in most browsers.

https://github.com/moodle/moodle/blob/38a3310c9272edfbac77ee4a8527e7838642072e/theme/boost/upgrade.txt#L93